lib/md/markdown.pest

```diff
@@ -13,7 +13,11 @@ bold2 = {"_"{2} ~ (words | (!"_"{2} ~ MARK))+ ~ "_"{2}}
 bold = { (bold1 | bold2) }
 singlebreak={NEWLINE{1}}
 linebreak=@{(" "{2} ~ NEWLINE | "<br>")}
-normal = { !(bold | italic) ~ (words | MARK)+}
+normal = {!(singlebreak | linebreak | list) ~ (bold | italic | words | MARK)+}
 emptyline = {NEWLINE{2}}
-paragraph = {((singlebreak | linebreak){0,1} ~ normal | bold | italic)+ ~ emptyline}
+paragraph = {((singlebreak | linebreak){0,1} ~ normal)+ ~ emptyline}
```

Comment:
Do we need to include the leading `singlebreak` and `linebreak` in the `paragraph`?

Reply:
Removed.